system/modules/core/forms/FormTextArea.php

```diff
@@ -144,7 +144,7 @@ public function __get($strKey)
 				break;
 
 			case 'value':
-				return specialchars(str_replace('\n', "\n", $this->varValue));
+				return str_replace('\n', "\n", $this->varValue);
```

Comment:
What if someone uses a custom textarea template? I don’t think it’s a good idea to change this behavior in a bugfix release.